homeassistant/components/logi_circle/__init__.py

```diff
@@ -91,14 +91,14 @@
 LOGI_CIRCLE_SERVICE_SNAPSHOT = vol.Schema(
     {
         vol.Optional(ATTR_ENTITY_ID): cv.comp_entity_ids,
-        vol.Required(ATTR_FILENAME): cv.template,
+        vol.Required(ATTR_FILENAME): str,
     }
 )
 
 LOGI_CIRCLE_SERVICE_RECORD = vol.Schema(
     {
         vol.Optional(ATTR_ENTITY_ID): cv.comp_entity_ids,
-        vol.Required(ATTR_FILENAME): cv.template,
+        vol.Required(ATTR_FILENAME): str,
```

Comment:
These services will need to be moved and registered as entity services in the logic_circle camera platform, to take advantage of the new entities template variable.
https://developers.home-assistant.io/docs/dev_101_services#entity-services

Reply:
Thanks, I didn't even have a look at that and just made the same changes as to the platform.
I think I'll remove the logi_circle changes for now. I'm not at all familiar with the integration and there are no tests outside of the config flow.